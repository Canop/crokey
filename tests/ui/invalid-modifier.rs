fn main() {
    crokey::key!(control-c);
}
