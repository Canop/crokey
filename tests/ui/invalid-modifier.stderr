error: unrecognized key code "control"
 --> tests/ui/invalid-modifier.rs:2:18
  |
2 |     crokey::key!(control-c);
  |                  ^^^^^^^
